tests/apitests/python/testutils.py

```diff
@@ -27,6 +27,8 @@
 USER_ROLE=dict(admin=0,normal=1)
 TEARDOWN = os.environ.get('TEARDOWN', 'true').lower() in ('true', 'yes')
 notary_url = os.environ.get('NOTARY_URL', 'https://'+harbor_server+':4443')
+DOCKER_USER = os.environ.get('DOCKER_USER', 'danfengliu')
+DOCKER_PWD = os.environ.get('DOCKER_PWD', 'Aa123456')
```

Comment:
@danfengliu I think it is better that you change your password.

Reply:
Thanks for your notice, we've planed to read secrete into environment variable, then pr can get the credential, the secrete setting is not ready yet. I will change my password soon.